README.md

```diff
@@ -147,6 +148,37 @@ To run tests:
 ```
 $ npm run test-all
 ```
+
+## Custom RJSF fields
+We have expanded the RJSF functionality with a base64 encoded file upload. It can be used by declaring a string field on a collection's JSON schema and specifying the base64file widget in the UI schema. Examples:
```

Comment:
```suggestion
We have expanded the RJSF functionality with a base64 encoded file upload. It can be used by declaring a `string` field on a collection's JSON schema and specifying the `base64file` widget in the UI schema. Examples:
```